systemtest/src/test/java/io/strimzi/systemtest/operators/user/UserST.java

```diff
@@ -369,6 +383,86 @@ void testCreatingUsersWithSecretPrefix(ExtensionContext extensionContext) {
         assertNull(kubeClient(namespaceName).getSecret(scramShaSecret.getMetadata().getName()));
     }
 
+    @ParallelNamespaceTest
+    void testTlsExternalUser(ExtensionContext extensionContext) {
+        final String namespaceName = StUtils.getNamespaceBasedOnRbac(NAMESPACE, extensionContext);
+        final String clusterName = mapWithClusterNames.get(extensionContext.getDisplayName());
+        final String userName = mapWithTestUsers.get(extensionContext.getDisplayName());
+        final String topicName = mapWithTestTopics.get(extensionContext.getDisplayName());
+
+        final AclRule writeRule = new AclRuleBuilder()
+            .withNewAclRuleTopicResource()
+                .withName(topicName)
+            .endAclRuleTopicResource()
+            .withOperation(AclOperation.WRITE)
+            .build();
+
+        final AclRule describeRule = new AclRuleBuilder()
+            .withNewAclRuleTopicResource()
+                .withName(topicName)
+            .endAclRuleTopicResource()
+            .withOperation(AclOperation.DESCRIBE)
+            .build();
+
+        // exercise (a) - create Kafka cluster with support for authorization
+        resourceManager.createResource(extensionContext, KafkaTemplates.kafkaEphemeral(clusterName, 1, 1)
+            .editSpec()
+                .editKafka()
+                    .withNewKafkaAuthorizationSimple()
+                        .withSuperUsers("CN=" + userName)
```

Comment:
Why do you set the user as super user? is that needed?

Reply:
We don't need that...